### ISSUE TITLE
[phpcs-fixer] "Generic.WhiteSpace.DisallowSpaceIndent" sniff issue with class DocBlock

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#278]
The `Generic.WhiteSpace.DisallowSpaceIndent` sniff mistakenly reports class DocBlock, which naturally have no indentation in the file, as `Tabs must be used to indent lines; spaces are not allowed` (code: `SpacesUsed`) error.

Example code:

``` php
<?php

defined('FULL_PATH') or die('restricted access!');

/**
 * Sample helper contains methods used in different classes
 *
 */
class SampleHelper extends kHelper
{



}
```

The 3 lines that start with `*` are being reported. I think that space before DocBlock asterisk should not be counted towards indentation at all.
